main.c

```diff
@@ -2,84 +2,173 @@
 #include <stdio.h>
 #include <unistd.h>
 #include <pthread.h>
+#include <termios.h>
 
+struct termios term; 
+ 
+void disableTerminalEcho ()
```

Comment:
Either make all functions with a space in between the name and brackets or none.